runtime/compiler/net/protos/compile.proto

```diff
@@ -47,14 +47,15 @@ message AnyData
 
 enum MessageType
    {
-   compilationCode = 0;
-   mirrorResolvedJ9Method = 1;
-   get_params_to_construct_TR_j9method = 2;
-   getUnloadedClassRanges = 3;
-   compilationRequest = 4; // type used when client sends remote compilation requests
-   compilationInterrupted = 5; // type used when client informs the server to abort the remote compilation
-   clientSessionTerminate = 6; // type used when client process is about to terminate
-   connectionTerminate = 7; // type used when client informs the server to close the connection
+   compilationEndCode = 0;
```

Comment:
I cannot decide on these names. Originally we had `compilationCode` which meant that the server was sending the compiled code back (at least that's how I interpret it). Would it make sense to keep `compilationCode` and add `compilationFailure` or something like that?

Reply:
I see. Thought `compilationCode` was meant to use at the end of the compilation. Sending the compiled code back to the client makes sense for `compilationCode`. Will update `compilationEndErrCode` to  `compilationFailure`.

Reply:
`compilationFailure` is already defined in `compilationErrorNames[]` in `rossa.cpp`. We could still define  `compilationFailure` as proto message type. Or to reduce confusion, how about `compilationErrors` ?

Reply:
If we always use it with `MessageType::` in front then there is no confusion.
Also, I saw some inclusion of rossa.h. Let me comment on that one as well